PayPal/src/main/java/com/braintreepayments/api/PayPalCreditFinancingAmount.java

```diff
@@ -22,12 +22,8 @@ private PayPalCreditFinancingAmount() {}
     static PayPalCreditFinancingAmount fromJson(JSONObject amount) {
         PayPalCreditFinancingAmount result = new PayPalCreditFinancingAmount();
 
-        if (amount == null) {
-            return result;
-        }
-
-        result.currency = Json.optString(amount, CURRENCY_KEY, null);
-        result.value = Json.optString(amount, VALUE_KEY, null);
```

Comment:
Are these changes needed (and in native checkout too)? It looks like this logic would already tolerate the fields not existing

Reply:
The change we made falls back to empty strings when possible. I'm not sure if it's needed or not for MXO, but the change was easy enough to prevent a merchant filing an inbound and having to tell the merchant it's deprecated etc.

Reply:
My concern with this is that now when the amount is null, the `PayPalCreditFinancingAmount` object returned will have the `currency` and `value` properties set to `""` rather than not set. That seems like unintended behavior

Reply:
Ok I see what you mean. I'll refactor the tests to expect `null` for those values.

Reply:
Made the update here: [8c95edc](https://github.com/braintree/braintree_android/pull/1059/commits/8c95edce819f3187c248d375bc0a9c41322e5013)